src/FormField/AutoComplete.php

```diff
@@ -1,457 +1,11 @@
 <?php
+/**
+ * @obsolete Backward compatibility.
```

Comment:
You should probably use `deprecated` instead of `obsolete`, `obsolete` is not officially supported in phpdoc - see https://docs.phpdoc.org/latest/references/phpdoc/index.html